management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/filter/FilteringSpec.java

```diff
@@ -10,24 +10,27 @@
 /**
  * Supports a subset of ServiceMonitorSpec's relabeling rules.
  *
- *             - source_labels: [subsystem, server]
- *               separator: "@"
- *               regex: "kata@webserver"
- *               action: "drop"
+ * - sourceLabels: [subsystem, server]
```

Comment:
Here is an example where your indent configs do not work. This automated indent messed up the comment's readability.